forestserviceprototype/specialuseform/models.py

```diff
@@ -3,49 +3,164 @@
 from django.contrib.auth.models import User
 from localflavor.us import models as localflavor
 
+
 # Create your models here.
 class Permit(models.Model):
```

Comment:
Just making sure that with these changes that the primary key is now `permit.permit_id`?